src/osd/scheduler/mClockScheduler.h

```diff
@@ -143,6 +147,18 @@ class mClockScheduler : public OpScheduler, md_config_obs_t {
     };
   }
 
+  static unsigned int get_io_prio_cut(CephContext *cct) {
+    if (cct->_conf->osd_op_queue_cut_off == "debug_random") {
+      srand(time(NULL));
```

Comment:
calling srand() every time?
and a more general question: is rand() the pref random generator for the mq? C++ has newer and better random generators.

Reply:
Yup, let's use std::random rather than srand.